### ISSUE TITLE
Cannot suppress 'package' keyword parse error, even with eslint-disable

### ISSUE THREAD

**Issue by OminousWater** [eslint/eslint#9483]
**Tell us about your environment**
MacOS 10.12.6

* **ESLint Version:**
4.9.0

* **Node Version:**
6.11.3

* **npm Version:**
3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?**
Both espree and esprima have the error

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

module.exports = { 
    "env": { 
        es6 : true 
    }, 
    "extends": "eslint:recommended", 
    "parserOptions": { 
        "ecmaVersion": 8, 
        "sourceType": "module" 
    }, 
    "rules": { 
        "indent": [ 
            "error", 
            "tab" 
        ], 
        "linebreak-style": [ 
            "error", 
            "unix" 
        ], 
        "quotes": [ 
            "error", 
            "double" 
        ], 
        "semi": [ 
            "error", 
            "always" 
        ] 
    } 
}; 


```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
/* Very simplified version of the actual source file, but it still demonstrates the problem */

/*eslint-disable*/
/* global package */

var name = package.name;

/*eslint-enable*/

```

**What did you expect to happen?**
I expected eslint to supress all errors (thanks to the /*eslint-disable*/ statement), but in particular, the error about parsing the 'package' keyword. I have tried many combinations of .eslintrc.js as well, setting the ecmascript environment to 6, enabling various language features and so on.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint throws up the following error (below is the error using espree, esprima throws up a slightly differently worded error, but to the same effect):

/private/tmp/temp/init-defaults.js
  6:12  error  Parsing error: The keyword 'package' is reserved

**Comment by not-an-aardvark** [comment]
Thanks for the report. This is working as intended; see https://github.com/eslint/eslint/issues/8992.

**Comment by OminousWater** [comment]
@not-an-aardvark Thanks for the impressively fast response! Is there a way to tell eslint to ignore the file completely (not even parse it)? As per the docs, I have tried applying the excludeFile rules to the config file as below, but it doesn't seem to stop eslint trying to parse the file (it is specifically the init-default.js file that we want to exclude, all other files are fine):

     "overrides": [
             {
                     "files": [ "*.js" ],
                     "excludedFiles": "init-defaults.js",
             }
     ]

**Comment by not-an-aardvark** [comment]
You should add the path to an [`.eslintignore` file](https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories). The `excludedFiles` option does something different; it refers to files which are excluded from a particular config override, not files that are excluded from linting in general.

**Comment by OminousWater** [comment]
Thanks v much, that's what I was looking for!
